ui/mainwindow.cpp

```diff
@@ -111,9 +121,9 @@ MainWindow::MainWindow(QWidget *parent): QMainWindow(parent), ui(new Ui::MainWin
 
   setWindowTitle(tr("GammaRay (%1)").arg(Endpoint::instance()->label()));
 
-  selectInitialTool();
```

Comment:
the change here means that upon selecting an initial tool, more tools get activated and that changes something? hmm that should be investigated.